README.md

```diff
@@ -29,11 +35,27 @@ From July 2023 `insightsengineering` packages are available on [r-universe](http
 # stable versions
 install.packages('teal.logger', repos = c('https://insightsengineering.r-universe.dev', 'https://cloud.r-project.org'))
 
+# install.packages("pak")
+pak::pak("insightsengineering/teal@*release")
+```
+
+Alternatively, you might also use the development version.
+
+```r
 # beta versions
 install.packages('teal.logger', repos = c('https://pharmaverse.r-universe.dev', 'https://cloud.r-project.org'))
+
+# install.packages("pak")
+pak::pak("insightsengineering/teal")
 ```
 
-See package vignettes `browseVignettes(package = "teal.logger")` for usage of this package.
+## Usage
+
+For usage of this package see package vignettes `browseVignettes(package = "teal.logger")` or package website [`insightsengineering.github.io/teal.logger`](https://insightsengineering.github.io/teal.logger/).
```

Comment:
```suggestion
To understand how to use this package, please refer to the [Getting Started](https://insightsengineering.github.io/teal.logger/latest-tag/articles/teal-logger.html) article, which provides multiple examples of code implementation.

Reply:
To be consistent with teal's README, we're pointing users to the exact URL from pkgdown website rather than `browseVignettes()`

Reply:
Just to comment on your comment. We need to be careful which endpoint we are using. You proposed `/latest-tag/` and this is ok. I have also seen `/main/` (or no ref which fallback to the main). I don't know what's the standard to be honest but I would encourage you to keep it consistent.

Please also don't use relative paths. It will be a broken path on e.g. r-universe, rdrr.io etc.

Speaking about the consistency this is what I am seeing currently on main:
![image](https://github.com/insightsengineering/teal.logger/assets/12943682/49197b21-fa63-4128-8609-4c4ebeccb495)
All of them don't have ref thus point to main. But these are no documentation entries.

Reply:
Thanks @donyunardi . I think `/latest-tag/` is fine, as this is the one that you get redirected to, the moment you enter `https://insightsengineering.github.io/teal.logger`. Also the `home` button on the page redirects to the `/latest-tag/` version as well.
<img width="684" alt="latest-tag" src="https://github.com/insightsengineering/teal.logger/assets/133694481/577b7b1f-bad1-4726-9218-a7c363033ef1">

Reply:
For the point that @pawelru mentioned, this is a bit more complex.
This url, has 3 different package versions depending on what you put as a subdir
https://insightsengineering.github.io/teal.logger/LICENSE-text.html
![image](https://github.com/insightsengineering/teal.logger/assets/133694481/0382fd88-245e-4426-b4bc-74518e64455b)

https://insightsengineering.github.io/teal.logger/main/LICENSE-text.html
![image](https://github.com/insightsengineering/teal.logger/assets/133694481/b6adec5d-7ec0-4988-b01f-9a7107fec84c)

https://insightsengineering.github.io/teal.logger/latest-tag/LICENSE-text.html
![image](https://github.com/insightsengineering/teal.logger/assets/133694481/b1919685-605a-4016-a39c-12cd0ddc27c1)


Unsure which one we should pick, and would appreciate your support on this. I would personally stick with everything pointing to the `latest-tag` as this will be consistent with what we think is released and tested and publically available for the stable version.

Reply:
I agree with your suggestion -> only `latest-tag` should be used.

Reply:
unified addresses in this commit 96babbadaf02f566dac1906cf208eae4a1b72d49

Reply:
Thank you. I think we probably need to do that for other repos. Can you please amend one of those docs issues for many repos to include that as well? Many thanks!

Reply:
Yes, just updated the description in the card for README changes https://github.com/insightsengineering/coredev-tasks/issues/462

Reply:
Thanks for the discussion both!